internal/controller/redis_connector.go

```diff
@@ -0,0 +1,76 @@
+package controller
+
+import (
+	"github.com/RedHatInsights/platform-receptor-controller/internal/platform/logger"
+	"github.com/RedHatInsights/platform-receptor-controller/internal/platform/utils"
+	"github.com/go-redis/redis"
+)
+
+var hostname string = utils.GetHostname()
+
+func addIndexes(client *redis.Client, account, nodeID, hostname string) {
+	client.SAdd("connections", account+":"+nodeID+":"+hostname) // get all connections
```

Comment:
We should probably move the logic to build the keys / values into methods so that they are not duplicated.